vignettes/manage.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "Manage lifecycle changes in functions you use"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Manage lifecycle changes in functions you use}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+options("lifecycle:::calling_package" = "tibble")
```

Comment:
This doesn't seem to actually be working as intended.

If I knit, I still see:

```
#> Warning: `data_frame()` is deprecated as of <NA> 1.1.0.
```